src/data/oppfolgingstilfelle/person/oppfolgingstilfellePersonQueryHooks.ts

```diff
@@ -16,7 +19,17 @@ const sortByDescendingStart = (
   });
 };
 
-const oppfolgingstilfellePersonQueryKeys = {
+const isInactive = (oppfolgingstilfelle: OppfolgingstilfelleDTO) => {
+  const today = dayjs(new Date());
+  const tilfelleEnd = dayjs(oppfolgingstilfelle.end);
+
+  return today.isAfter(
+    tilfelleEnd.add(ARBEIDSGIVERPERIODE_DAYS, "days"),
+    "date"
+  );
+};
+
```

Comment:
Tror du må sette tidspunkter på `today` til 00:00 e.g. med setHours(0,0,0,0) for at dette skal bli konsistent med koden i Isdialogmote.

Reply:
Tror det skal være greit når jeg sender inn "date" som unit til den isAfter-metoden (ref: https://day.js.org/docs/en/query/is-after). Kunne droppet å bruke dayjs og heller sammenlignet Date-objektene sikkert, men da måtte man nok gjort som du sier for å ignorere klokkeslett ja.